### PR TITLE
Core database models and migrations

### PR DESCRIPTION
**Pull request by Frederick-Teye**
Created the following:
1. Custom User model with student/lecturer roles
2.  Course model
3. Review model
4. Grade model for lecturer assessments
5. Set up model relationships and foreign keys
6. Implement model validation rules
7. Add database constraints for mutual reviews
8. Create migrations

### CODE REVIEW

File: accounts/admin.py
[the remark this answers is not in the document]
Comment (Frederick-Teye) [reply accounts/admin.py:1]: Okay bro, thank you...

**Comment by Frederick-Teye** [comment]
I have pushed the new changes.

File: .vscode/settings.json
Comment (JohananOppongAmoateng) [inline .vscode/settings.json:1]: Put this file in a gitignore file

File: peer_review/settings.py
Comment (JohananOppongAmoateng) [inline peer_review/settings.py:1]: U still haven't put the secret key as an env variable

**Review by JohananOppongAmoateng — CHANGES_REQUESTED** [review]

**Comment by Frederick-Teye** [comment]
Everything is done now, please check and give me feedback

**Comment by Frederick-Teye** [comment]
I'm seeing some red reds over here about some checks were not successful

**Pull request merged**